setup.py

```diff
@@ -300,8 +335,8 @@ def build_extension(self, ext):
 def run_setup(with_extensions):
 
     if with_extensions:
-        ext_modules = (blosc_extension() + zstd_extension() + lz4_extension() +
-                       compat_extension() + vlen_extension())
+        ext_modules = (blosc_extension() +  lz4_extension() + # zstd_extension() +
+                       zfp_extension() + compat_extension() + vlen_extension())
```

Comment:
Is there a reason why you commented out `zstd_extension()`?

Reply:
Looks like this got added back in subsequent revisions. Marking as resolved. Though please adjust if I've missed something.